randovania/game_description/game_migration.py

```diff
@@ -299,6 +299,14 @@ def _migrate_v27(data: dict, game: RandovaniaGame) -> None:
                     node["hint_features"] = []
 
 
+def _migrate_v28(data: dict, game: RandovaniaGame) -> None:
+    for world in data["worlds"]:
+        for area in world["areas"].values():
+            for node in area["nodes"].values():
+                if node["node_type"] == "pickup" and node.get("pickup_type") == "Energy Transfer Module":
+                    node["pickup_type"] = "Nothing"
+
```

Comment:
Can you explain what your thought process was when implementing this function?
I'm very confused on where you got all of these from :sweat_smile: . They're also wrong as they're making the tests fail.